db/1.9/modules/ROOT/edits/database-connector-query-single.adoc

```diff
@@ -0,0 +1,215 @@
+== Query a Single Record from a Database Example - Mule 4
+
+The *Query single* operation enables you to execute an SQL query that returns a single record (row) from a database and access the result payload without any additional transformations. This operation does not stream any of the returned fields. When the query is executed, all the values of the returned row are read into memory and returned in the payload. You must be careful, because if your row is too large, you could run out of memory.
```

Comment:
```suggestion
The *Query single* operation enables you to execute an SQL query that returns a single record (row) from a database and access the result payload without any additional transformations. This operation does not stream any of the returned fields. When the query is executed, all the values of the returned row are read into memory and returned in the payload. You must be careful, however, because if your row is too large, you could run out of memory.
```

Reply:
I wonder if we should start a new paragraph after "payload" and include both recommendations: to ensure that rows are not too long (to avoid memory loss) and to use pooling (to avoid insufficient connections).